app/assets/stylesheets/application.css

```diff
@@ -46,14 +47,18 @@
 
 .img {
   margin-top: 10px;
-  width: 20%;
+  width: 10%;
+  padding: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid black;
 }
 
 .main {
   border: 2px solid black;
   padding: 10px;
-  flex: 10;
-  width: 20vw;
+  width: 70%;
 }
 
```

Comment:
- I would recommend using scss file rather than CSS for the rails project, for more clarification check which better CSS vs scss in the rails project.